x-pack/apm-server/aggregation/spanmetrics/aggregator.go

```diff
@@ -264,13 +271,18 @@ func (a *Aggregator) processDroppedSpanStats(event *model.APMEvent, dss model.Dr
 		dss.DestinationServiceResource,
 		dss.ServiceTargetType,
 		dss.ServiceTargetName,
+
+		// BUG(axw) dropped span statistics do not contain span name.
+		// Capturing the service name requires changes to Elastic APM agents.
```

Comment:
While aggregating metrics in APM agents would be feasible and more accurate, I'm not quite sure if it's worth the additional memory overhead in the agents. Exit spans that accidentally create high cardinality may also be more impactful in APM Agents.

Reply:
Yeah fair point. Perhaps we should just document that the operation stats don't include dropped spans.